src/frontend/package.json

```diff
@@ -95,7 +97,6 @@
     "webpack-cli": "3.3.12",
     "whatwg-fetch": "3.4.1",
     "xhr-mock": "2.5.1",
-    "xstate": "4.13.0",
```

Comment:
You rewrote everything made with this library in the `CourseRunEnrollment` component. Why this choice ?

Reply:
I don't want a finish state machine since I would like to reset state if user session changed. Furthermore, it was one of the heaviest library used in the project and it was used only at one place. Finally it is easily replacable with react builtin hooks.